app/components/shared/Hotkey.vue.ts

```diff
@@ -45,14 +45,15 @@ export default class HotkeyComponent extends TsxComponent<HotkeyProps> {
     const binding = this.bindings[index];
 
     const key =
+      /*eslint-disable*/
       event instanceof MouseEvent
         ? {
             1: 'MiddleMouseButton',
             3: 'X1MouseButton',
             4: 'X2MouseButton',
           }[event.button]
         : event.code;
-
+      /*eslint-enable*/
```

Comment:
Will the `eslint-disable` run until the end of the file if this is accidentally removed?  Seems a little dangerous, but hopefully eslint throws an error if you omit the closing tag?

Reply:
Unfortunately, it doesn't throw an error